lib/Menu.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/order_confirmation.rb'
+
+class Menu
+
+  attr_reader :basket
+
+  def initialize(menu = [], basket = Basket.new)
```

Comment:
use of array assignment to menu so smart! 😊

Reply:
thanks